tofhir-server/src/main/scala/io/tofhir/server/util/CsvUtil.scala

```diff
@@ -79,34 +80,43 @@ object CsvUtil {
         }
       }
 
+
     // here an example existingContent looks like:
     // [
```

Comment:
The structure of the example is confusing. Could you please change them to csv like: 

header1,header2,header3
value1,value2,value3
value4,value5,value6

which is more readable. Update the other examples as well.